ElementX/Sources/Screens/ReportContent/View/ReportContentScreen.swift

```diff
@@ -0,0 +1,109 @@
+    
+    private var horizontalPadding: CGFloat {
+        horizontalSizeClass == .regular ? 50 : 16
+    }
+
+    var body: some View {
+        ScrollView {
+            mainContent
+                .padding(.top, 50)
+                .padding(.horizontal, horizontalPadding)
+        }
+        .scrollDismissesKeyboard(.immediately)
+        .background(Color.element.formBackground.ignoresSafeArea())
+        .navigationTitle(ElementL10n.reportContent)
+        .navigationBarTitleDisplayMode(.inline)
+        .toolbar { toolbar }
+        .interactiveDismissDisabled()
+    }
+
+    /// The main content of the view to be shown in a scroll view.
+    var mainContent: some View {
+        VStack(alignment: .leading, spacing: 24) {
+            Text(ElementL10n.reportContentInfo)
+                .font(.element.body)
+                .foregroundColor(Color.element.primaryContent)
+            descriptionTextEditor
+        }
+    }
+
+    @ViewBuilder
+    private var descriptionTextEditor: some View {
+        ZStack(alignment: .topLeading) {
+            RoundedRectangle(cornerRadius: 14, style: .continuous)
+                .fill(Color.element.formRowBackground)
+
+            TextEditor(text: $context.reasonText)
+                .tint(.element.brand)
+                .padding(.horizontal, 10)
+                .padding(.vertical, 4)
+                .cornerRadius(14)
+                .accessibilityIdentifier(A11yIdentifiers.reportContentScreen.reason)
+                .scrollContentBackground(.hidden)
+
+            if context.reasonText.isEmpty {
+                Text(ElementL10n.reportContentCustomHint)
+                    .font(.element.body)
+                    .foregroundColor(Color.element.secondaryContent)
+                    .padding(.horizontal, 16)
+                    .padding(.vertical, 12)
+                    .allowsHitTesting(false)
+            }
+
+            RoundedRectangle(cornerRadius: 14, style: .continuous)
+                .stroke(Color.element.quaternaryContent)
+        }
+        .frame(maxWidth: .infinity)
+        .frame(height: 220)
+        .font(.body)
+    }
+
+    @ToolbarContentBuilder
+    private var toolbar: some ToolbarContent {
+        ToolbarItem(placement: .cancellationAction) {
+            Button(ElementL10n.actionCancel) {
+                context.send(viewAction: .cancel)
+            }
+        }
+
+        ToolbarItem(placement: .confirmationAction) {
+            Button(ElementL10n.actionSend) {
+                context.send(viewAction: .submit)
+            }
+            .accessibilityIdentifier(A11yIdentifiers.bugReportScreen.send)
+        }
+    }
+}
+
+// MARK: - Previews
+
+struct ReportContent_Previews: PreviewProvider {
+    static var previews: some View {
+        Group {
+            let viewModel = ReportContentViewModel(itemID: "", roomProxy: MockRoomProxy(displayName: nil))
+            ReportContentScreen(context: viewModel.context)
+        }
+    }
```

Comment:
Sorry, I've tidied most of these up in #651, so this is just to keep consistency.
```suggestion
    static let viewModel = ReportContentViewModel(itemID: "", roomProxy: MockRoomProxy(displayName: nil))
    
    static var previews: some View {
        ReportContentScreen(context: viewModel.context)
    }
```